### DISCUSSION TITLE
How to handle missing data intervals?

### DISCUSSION THREAD

**Discussion by cosminlinca** [patrykandpatrick/vico#926] [Questions] [answered]
### Question

Hello, how can I handle missing intervals in a line chart using the library? For example, given the following data:

```
val sampleLineChartData = ChartData(
    seriesValues = listOf(80, 110, null, null, null, 70, 85),
    bottomAxisValues = listOf("Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul")
)
```

I want the chart to display no data (a gap) for the null values in March, April, and May, and then continue the line starting from June. Is there a way to achieve this?

### Vico version(s)

2.0.0-alpha.28

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment, marked as answer]
Hello, @cosminlinca. You have to use multiple series linked to the same `LineCartesianLayer.Line`.
